tests/test/codegen/test_machine_code_generator.cpp

```diff
@@ -112,20 +112,15 @@ TEST_CASE_METHOD(CodegenTestFixture,
     SECTION("WAMR codegen")
     {
         conf.wasmVm = "wamr";
+        objectFileA = "/tmp/obj/demo/hello/function.aot";
+        objectFileB = "/tmp/obj/demo/echo/function.aot";
+    }
 
-        SECTION("Non-SGX")
-        {
-            objectFileA = "/tmp/obj/demo/hello/function.aot";
-            objectFileB = "/tmp/obj/demo/echo/function.aot";
-        }
-
-        SECTION("SGX")
-        {
-            objectFileA = "/tmp/obj/demo/hello/function.aot.sgx";
-            objectFileB = "/tmp/obj/demo/echo/function.aot.sgx";
-            msgA.set_issgx(true);
-            msgB.set_issgx(true);
-        }
+    SECTION("SGX codegen")
+    {
+        conf.wasmVm = "sgx";
```

Comment:
Can you double check that there's a `FaasmConfigTestFixture` somewhere in the class hierarchy for this test? We just want to make sure that the config is reset at the end of each test.

Reply:
There is an `S3TestFixture` that resets the config :+1: